docs-2.0/3.ngql-guide/9.space-statements.md/1.create-space.md

```diff
@@ -40,7 +40,7 @@ You can set four optional options for a new graph space:
 
 * `vid_type`
 
-    Specifies the data type of vertex IDs (VIDs) in a graph space. In Nebula Graph 2.0.0-alpha, only `fixed_string(N)` is valid, indicating VIDs must be string type. `N` represents the maximum length of the VIDs. You must set the `fixed_string` to a positive integer. The default value is 8. If your VID length is greater than the maximum VID length, Nebula Graph throws an error.
+    Specifies the data type of vertex IDs (VIDs) in a graph space. The default value is `FIXED_STRING(8)`. This shows that the default VID data type is fixed `string` and the default length of `8`. In Nebula Graph 2.0.0-RC, only `FIXED_STRING(N)` and `INT64` are supported. `N` represents the maximum length of the VIDs. You must set the `fixed_string` to a positive integer. If your VID length is greater than the maximum VID length, Nebula Graph throws an error. To set the integer VID for vertices, specify the `vid_type` to `INT64`.
```

Comment:
> This shows that the default VID data type is fixed `string` and the default length of `8`.
(Readability) fixed `string` ==> `FIXED_STRING` or fixed string
(Grammar) the default length of `8` ==> the default length is 8.